docs/platforms/godot/data-management/data-collected.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. Here's a list of data categories the Sentry Native SDK collects:
+
+## User Information
+
+By default, the Sentry SDK doesn't send any information about users, such as email address, user ID, or username. However, if the <PlatformLink to="/configuration/options/#send-default-pii">`send_default_pii` option</PlatformLink> is enabled, Sentry backend services will infer the user's IP address based on the incoming request.
+
+You can also set user information from code:
+
+```GDScript
+var user := SentryUser.new()
+user.id = "custom_id"
+user.email = "bob@example.com"
+user.username = "bob"
+user.ip_address = "127.0.0.1"
+SentrySDK.set_user(user)
+```
+
+## Device Information
+
+The Sentry SDK collects information about the device, such as the name, version and build of your operating system or Linux distribution. This information is sent to Sentry by default.
+
+## Screenshots
+
+The <PlatformLink to="/enriching-events/screenshots">screenshot feature</PlatformLink> is disabled by default. If you choose to enable this feature in options, any screenshots captured may contain sensitive data visible in the application at the time of the error.
+
+## Godot Log File
+
+By default, the Sentry SDK automatically attaches the Godot log file to events it sends. Since this log file contains all messages printed in your code and by the engine itself, it may inadvertently include sensitive information. To enhance privacy protection, you can disable the <PlatformLink to="/configuration/options/#attach-log">`attach_log` option</PlatformLink> in the Godot Project Settings.
+
+## Thread Stack Information
+
+At the time of a crash, the stack of each thread is collected and sent to Sentry as part of the Minidump snapshot for `crashpad` backend. This information is sent to Sentry by default, but dropped after processing the event in the backend.
```

Comment:
Note: We're linking out to `/native` since we don't have that page in the Godot docs.

We should bring that page over ideally as an include so we can add to Unreal, Unity, Godot, Native, whatever else that makes sense.

```suggestion
At the time of a crash, the stack of each thread is collected and sent to Sentry through a minidump. This information is sent to Sentry by default, but dropped after processing the event. [You can configure Sentry to persist the minidump as attachment](/platforms/native/data-management/store-minidumps-as-attachments/).
```